bsconfig.json

```diff
@@ -19,5 +22,6 @@
       "dir": "benchmark",
       "type": "dev"
     }
-  ]
+  ],
+  "uncurried": false
```

Comment:
If an application which uses the package has `uncurried: true`, will the compiler ignore the package's config, or not?

Reply:
Good question, I'm really not sure! 

I'd guess so, because it would be chaos otherwise.

Maybe moving to uncurried by default should be a major version bump that only supports rescript v11 upwards.

Reply:
Yeah, it's an interesting question. Maybe it's a good idea to move to uncurried by default for v11.

Reply:
I've tested it out. Yes, the packages bsconfig is ignored. So, I've added the `@uncurried` on top of my package file, which is not being overwritten.